packages/components/src/modifiers/hds-advanced-table-cell/keyboard-navigation.ts

```diff
@@ -124,6 +50,7 @@ export const handleGridCellKeyPress = (
         enableFocusTrap();
       }
     } else if (event.key === 'ArrowRight' || event.key === 'ArrowLeft') {
+      event.preventDefault();
```

Comment:
Leaving a comment here, for posterity, that we're introducing these to prevent the page from moving when reaching a boundary cell (I suspect, correct me if I'm wrong).

Reply:
That is the case, @MelSumner requested that behavior.